app/controllers/form_controller.rb

```diff
@@ -15,7 +15,7 @@ def submit_form
         redirect_to(successful_redirect_path)
       else
         mandatory_questions_with_no_response.map do |question|
-          @log.errors.add question.id.to_sym, question.unanswered_error_message
+          @log.errors.add question.id.to_sym, question.unanswered_error_message, category: :not_answered
```

Comment:
this category has been added to all non-setup errors that are triggered when a compulsory field is blank, so that later on we can only show the user answers that will be deleted **and are not already blank** in the error count

Reply:
would :blank make sense here instead?